project/AkkaBuild.scala

```diff
@@ -125,6 +125,19 @@ object AkkaBuild extends Build {
     dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm", slf4j % "test->compile")
   ) configs (MultiJvm)
 
+  lazy val clusterTools = Project(
+    id = "akka-cluster-tools",
+    base = file("akka-cluster-tools"),
+    dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm")
+  ) configs (MultiJvm)
+  
+  lazy val clusterSharding = Project(
+    id = "akka-cluster-sharding",
+    base = file("akka-cluster-sharding"),
+    dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm", 
+        persistence % "compile;test->provided", clusterTools)
```

Comment:
When I try other things it fails running tests with "missing or invalid dependency detected while loading class file 'LeveldbIdMapping.class'."

I checked the pom and it looks correct, nothing about level db or provided.